src/ReactSelectize.ls

```diff
@@ -29,6 +29,7 @@ module.exports = create-class do
         # class-name :: String
         delimiters: []
         disabled: false
+        maxLength: 255
```

Comment:
Why make a default limit? this is not backward compatible (i.e. Some usages may need more than 255 characters).